src/cctz_benchmark.cc

```diff
@@ -104,6 +104,7 @@ const char RFC1123_no_wday[] = "%d %b %Y %H:%M:%S %z";
 // A list of known time-zone names.
 // TODO: Refactor with src/time_zone_lookup_test.cc.
 const char* const kTimeZoneNames[] = {
+    // clang-format off
```

Comment:
Do you think we should indent the list another level to match the indentation of the comment/directive?